pom.xml

```diff
@@ -344,12 +356,15 @@
         <property>
           <name>!pdfjt-lm</name>
         </property>
+        <file>
+          <missing>${basedir}/.use-pdfjt-lm</missing>
+        </file>
       </activation>
       <dependencies>
         <dependency>
-          <groupId>com.adobe.pdfjt</groupId>
```

Comment:
Nice removal of redundant dependency here because PDFJT is a dependency of Talkeetna.